test/integration/windows/button/index.jsx

```diff
@@ -236,8 +236,8 @@ if (onRender) {
     click() {
       getElement(".paypal-button", document).click();
     },
-    hoverMessage(): ZalgoPromise<void> | void {
-      return window.xprops.onMessageHover();
+    hoverMessage({ merchantID }): ZalgoPromise<void> | void {
+      return window.xprops.onMessageHover({ merchantID });
```

Comment:
It looks like we're no longer consuming the `merchantID` from the hover callback. Are these changes able to be reverted?